app/routers/views.py

```diff
@@ -91,16 +91,10 @@ async def logout(request: Request, response: Response):
     I usually open a new tab/window to reset basic auth. We likely
     need a logout button to be handled somehow in javascript.
     """
-    response.delete_cookie("basic")
-    response.delete_cookie("bearer")
-    response.delete_cookie("access_token")
-    data = helpers.get_page("index.md")
-    return templates.TemplateResponse(
-        "index.html",
-        {
-            "request": request,
-            "data": data,
-        },
+    raise HTTPException(
```

Comment:
I think we still want to give the user a good experience here - the initial intention to redirect somewhere was to do that. Right now this logout function seems to just raise an exception but I'm not sure it does anything?

Reply:
For better user experience, I think we should start looking into another authentication method since Basic Auth is not designed to handle logout functionality (here's the rationale I found: https://stackoverflow.com/a/233551) but I'm open to any suggestion!